templates/organization/user/_list.html.twig

```diff
@@ -18,6 +19,11 @@
                 <td>{{ user.identificationNumber }}</td>
                 <td>{{ user.firstName }}</td>
                 <td>{{ user.lastName }}</td>
+                <td>
+                    {% for skill in user.skillSet -%}
+                        <span class="badge badge-{{ skill in importantSkills ? 'primary' : 'secondary' }}">{{ skill }}</span>
```

Comment:
You can maybe use something like `{{ skill | skillBadge }}` or `{{ skillBadge(skill) }}` here and in all places where we display those skills ?